scripts/install-all-subpackages.ts

```diff
@@ -0,0 +1,27 @@
+import { execSync, type SpawnSyncReturns } from "node:child_process";
+import path from "node:path";
+import { findDirWithFileRecursive } from "./find.js";
+
+export async function installAllSubpackages(
+  filename: string,
+  command: string
+): Promise<void> {
+  for (const project of await findDirWithFileRecursive(filename)) {
+    if (project.split(path.sep).length < 2) continue;
+    try {
+      console.log(`+ ${command}`);
```

Comment:
Currently, the commands include the `--cwd` or `--prefix` parameter, which allows to distinguish directories, so you'll want to add it to the log statement.

```suggestion
      console.log(`+ ${command} [${project}]`);
```